### ISSUE TITLE
Running DiaNN under Wine

### ISSUE THREAD

**Issue by caldodge** [vdemichev/DiaNN#370]
I've been trying this with Wine 6.8 without success on CentOS 7.5.
Thinking it might be a library issue, I installed the WIndows version with Wine 7.5 on Fedora 35.

The GUI starts, but the conversion (input wiff files) fails.  I see the following messages:

100:fixme:commdlg:IServiceProvider_fnQueryService Interface {e07010ec-bc17-44c0-97b0-46c7c95b9edc} requested from unknown service {e07010ec-bc17-44c0-97b0-46c7c95b9edc}
0100:fixme:shell:ViewModeToListStyle ViewMode 0 not implemented
0100:fixme:shell:IShellBrowser_fnSendControlMsg stub, 000000000E8B3120 (2, 1026, a003, 0, 000000000041C018)
0100:fixme:shell:IShellBrowser_fnSendControlMsg stub, 000000000E8B3120 (2, 1026, a004, 0, 000000000041C018)
0100:fixme:shell:IShellBrowser_fnSendControlMsg stub, 000000000E8B3120 (2, 1025, a003, 1, 000000000041C018)
0100:fixme:shell:IShellBrowser_fnSendControlMsg stub, 000000000E8B3120 (2, 1025, a004, 1, 000000000041C018)
0100:fixme:nstc:NSTC2_fnSetControlStyle2 mask & style (0x00000004) contains unsupported style(s): 0x00000004
0100:fixme:dwmapi:DwmExtendFrameIntoClientArea (000000000001005E, 000000000041A9C0) stub
0100:fixme:dwmapi:DwmExtendFrameIntoClientArea (000000000001005E, 000000000041BEC0) stub
0168:fixme:msvcp:_Locinfo__Locinfo_ctor_cat_cstr (000000000011EEE0 1 C) semi-stub
0168:fixme:msvcp:_Locinfo__Locinfo_ctor_cat_cstr (000000000011EC10 1 C) semi-stub
0168:fixme:msvcp:_Locinfo__Locinfo_ctor_cat_cstr (000000000011F480 1 C) semi-stub
0168:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION

Unhandled Exception:
System.IO.FileNotFoundException: Could not load file or assembly 'Clearcore2.Data.AnalystDataProvider, Version=3.0.0.0, Culture=neutral, PublicKeyToken=2a79e0d8fd2e4eca' or one of its dependencies.
File name: 'Clearcore2.Data.AnalystDataProvider, Version=3.0.0.0, Culture=neutral, PublicKeyToken=2a79e0d8fd2e4eca'
  at (wrapper native-to-managed) <Module>.diann_wiff_load(sbyte modopt(System.Runtime.CompilerServices.IsSignUnspecifiedByte)*,byte,int)
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.FileNotFoundException: Could not load file or assembly 'Clearcore2.Data.AnalystDataProvider, Version=3.0.0.0, Culture=neutral, PublicKeyToken=2a79e0d8fd2e4eca' or one of its dependencies.
File name: 'Clearcore2.Data.AnalystDataProvider, Version=3.0.0.0, Culture=neutral, PublicKeyToken=2a79e0d8fd2e4eca'
^Z

Are there any guidelines on running under Wine, like perhaps undocumented dependencies?

**Comment by vdemichev** [comment]
Were all the Sciex libraries copied to the folder with diann.exe?

**Comment by caldodge** [comment]
No.  Are they part of the install?  Or is a separate installer needed?

**Comment by vdemichev** [comment]
Please see https://github.com/vdemichev/DiaNN#raw-data-formats

**Comment by caldodge** [comment]
OK, I now get the following error:

et to 6

1 files will be processed
[0:00] Loading run Z:\home\calvin\diann-test\PDAY_160411_SWATH_6600_Test_Sample6_1X_Run1.wiff
ERROR: cannot read the .wiff file. Perhaps the respective .wiff.scan file is absent or corrupted?
ERROR: cannot read the .wiff file. Perhaps the respective .wiff.scan file is absent or corrupted?
ERROR: cannot read the .wiff file. Perhaps the respective .wiff.scan file is absent or corrupted?
ERROR: cannot read the .wiff file. Perhaps the respective .wiff.scan file is absent or corrupted?
ERROR: cannot read the .wiff file. Perhaps the respective .wiff.scan file is absent or corrupted?


Is a wiff.scan file required for each wiff file?  If so, I'll discus it with the user.

**Comment by vdemichev** [comment]
Yes, .wiff and .wiff.scan should be in the same folder

**Comment by caldodge** [comment]
My apologies - since we support users, but don't use DiaNN, we weren't aware of the need for the scan files.  We will be testing with those shortly.

**Comment by caldodge** [comment]
The conversion is failing with raw files, after installing the specific Thermo reader program.  Has this been demonstrated to work in wine?  Are any extra steps required, as were needed with pwiz dlls?

**Comment by caldodge** [comment]
I am not able to download MSFileReader (don't have an existing relationship with Thermo), so the customer did.  According to file details, it's version 3.1.0.  Does that mean the customer downloaded the wrong version?

**Comment by vdemichev** [comment]
Yes, need exactly the correct version. Don't need a relationship with Thermo, anyone can create a free account and download.

**Comment by caldodge** [comment]
Thanks.  That version does appear to do a conversion in Wine.  I'll have the customer try it to confirm the results look good.

-rwxrwxrwx 1 calvin calvin 482070431 Feb 24 00:57 220222_JeffriesLewis_007_CarolineJeffries_NativePlasma_a2_P1004.raw
-rw-r--r-- 1 calvin calvin 113808820 May  5 16:17 220222_JeffriesLewis_007_CarolineJeffries_NativePlasma_a2_P1004.raw.dia
